### PR TITLE
Pypi

### PR DESCRIPTION
**Pull request by dtamayo**
I changed the version back to the current one (2.11.1), so change it to whatever you like.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -7,6 +7,9 @@ include src/integrator_hybrid.c
 include src/integrator.c
 include src/gravity.c
 include src/collision.c
+include src/communication_mpi.c
```
Comment (hannorein) [inline MANIFEST.in]: Do you really need those?

Comment (dtamayo) [inline MANIFEST.in]: I couldn’t tell why some were included and not others, and I think in part of my testing, I added them all to make sure they were all added to the tarball on pypi.  I think you can just remove them all if you don’t want them (except rebound/rebound.h)

Comment (hannorein) [inline MANIFEST.in]: They are not needed as Python doesn't work together with MPI or OpenGL.

Comment (dtamayo) [inline MANIFEST.in]: That makes sense.

**Pull request merged**